api-model/src/main/java/io/enmasse/admin/model/v1/AddressSpacePlanSpec.java

```diff
@@ -81,6 +81,6 @@ public void setAddressPlans(List<String> addressPlans) {
     }
 
     public List<String> getAddressPlans() {
-        return Collections.unmodifiableList(addressPlans);
+        return addressPlans;
```

Comment:
This should not be modifyable

Reply:
this cause NPE

Reply:
@lulf that is the reson why I remove that